src/chainparams.h

```diff
@@ -92,7 +92,6 @@ class CChainParams
     int FutureBlockTimeDrift(const int nHeight) const;
     bool IsValidBlockTimeStamp(const int64_t nTime, const int nHeight) const;
 
-    CAmount MaxMoneyOut() const { return nMaxMoneyOut; }
```

Comment:
Should remove also the definition of the class variable `nMaxMoneyOut` at line 197.

Reply:
good catch, done